src/Chapter1.hs

```diff
@@ -615,7 +638,12 @@ Try to introduce variables in this task (either with let-in or where) to avoid
 specifying complex expressions.
 -}
 
-sumLast2 n = error "sumLast2: Not implemented!"
+sumLast2 :: Int -> Int
+sumLast2 n = 
+  let absN = abs n
+      lastNum = absN `mod` 10
+      secondLastNum = ((absN `mod` 100) - lastNum) `div` 10
```

Comment:
I think that you don't need to subtract `lastNum` in here 🙂 

Also, see that the `lastNum` look a bit familiar? It is the `lastDigit` from the previous task 👍🏼